src/firebase.js

```diff
@@ -26,10 +27,59 @@ if (window.location.hostname === "localhost") {
   db.useEmulator("localhost", 8080);
 }
 
+const addConfigToFirebase = (participantID, studyID, startDate, config) => {
+  console.log("Adding config to Firebase")
+  db.collection(RESPONSE_COLLECTION_NAME)
+    .doc(studyID)
+    .collection("participants")
+    .doc(participantID)
+    .collection("data")
+    .doc(startDate)
+    .update({ config: config });
+};
+
+const getFirestoreConfig = (studyID, docName) => {
+  return db
+    .collection(REGISTERED_COLLECTION_NAME)
+    .doc(studyID)
+    .collection("config")
+    .doc(docName)
+    .get()
+    .then((doc) => {
+      if (doc.exists) {
+        console.log(`Doc ${docName} exists`)
+        return JSON.parse(doc.data().config);
+      } else {
+        console.log(`Document ${docName} does not exist`);
+        return false;
+      }
+    })
+    .catch((error) => console.log("Error in getting config:", error));
+};
+
+/**
+ * Gets the config object for the logged-in participant, or uses the default for the study. The config object
+ * is a string.
+ * @param {string} studyID The study ID specified at login.
+ * @param {string} participantID The logged in participant ID.
+ */
+const firestoreConfig = async (studyID, participantID) => {
```

Comment:
or maybe `getParticipantFirestoreConfig`? or rename the other one to `getFirestoreConfigObject` or something? names are hard.